sys/net/routing/rpl/Makefile

```diff
@@ -1,8 +1,16 @@
-MODE = $(shell grep "RPL_DEFAULT_MOP RPL_NON_STORING_MODE" ../../include/rpl/rpl_config.h)
-ifneq (,$(MODE))
-	DIRS += rpl_nonstoring
+MODE = $(shell grep "\#define $(RPL_MOP)" ../../include/rpl/rpl_config.h | sed 's/\#define \s*//g')
```

Comment:
Hm, I somehow don't like this. For me it would be much cleaner to simply let it to the user (aka application developer) to specify the MOP number as a parameter and get rid of this whole grep stuff here.